src/main/drivers/pwm_output_dshot.c

```diff
@@ -138,7 +140,7 @@ void pwmDshotMotorHardwareConfig(const timerHardware_t *timerHardware, uint8_t m
     timerOCInit(timer, timerHardware->channel, &TIM_OCInitStructure);
     timerOCPreloadConfig(timer, timerHardware->channel, TIM_OCPreload_Enable);
     motor->timerDmaSource = timerDmaSource(timerHardware->channel);
-    dmaMotorTimers[timerIndex].timerDmaSources |= motor->timerDmaSource;
```

Comment:
For completeness, it may be better to clear corresponding source here. (It is initialized to zero and cleared in pwmCompleteDshotMotorUpdate, but better be safe)